Source/WebCore/css/html.css

```diff
@@ -1453,12 +1453,6 @@ applet, embed, object, img {
     }
 }
 
-/* Default support for "Smart Invert" where all content color except media is inverted. */
-/* https://drafts.csswg.org/mediaqueries-5/#inverted */
-@media (inverted-colors) {
-    img:not(picture>img), picture, video { filter: invert(100%); } /* Images and videos double-inverted. */
-}
-
```

Comment:
Removing this would break support for a major accessibility feature on multiple platforms.

Reply:
Would you be able to share this concern on https://github.com/w3c/csswg-drafts/issues/9674 where the CSSWG resolved to move this UA style to an author suggestion?